src/DeviceStatusPage/DeviceLiveEventsCard.tsx

```diff
@@ -260,12 +260,25 @@ const astarteDeviceEventBody = (event: AstarteDeviceEvent) => {
     );
   }
   if (event instanceof AstarteDeviceErrorEvent) {
+    const metadata = JSON.parse(JSON.stringify(event.metadata));
```

Comment:
event metadata is already an object, why are we forcing it to a string and then parsing it?